src/collective/cover/tests/test_layout_resize.robot

```diff
@@ -0,0 +1,48 @@
+*** Settings ***
+
+Resource  cover.robot
+Library  Remote  ${PLONE_URL}/RobotRemote
+
+Suite Setup  Open Test Browser
+Suite Teardown  Close all browsers
+
+*** Variables ***
+
+${basic_tile_name} =  "collective.cover.basic"
+${tile_class} =  div.cover-tile
+
+*** Test cases ***
+
+Test Columns-Titles and Resize
+
+    Enable Autologin as  Site Administrator
+    Go to Homepage
+
+    Create Cover  Save-Cover  Test-cover
+    Open Layout Tab
+
+    Add Tile  ${basic_tile_name}
+    Click Element  css=i.resizer
+    Wait until element is visible  id=ui-id-1
+    Drag And Drop By Offset  css=a.ui-slider-handle.ui-state-default.ui-corner-all  -30  0
+    Click Element  css=.ui-dialog-titlebar-close
+
+    Save Cover Layout
+
+    Click Link  link=Compose
+    Open Layout Tab
+
+    ${data-column-size}=  Get Element Attribute  xpath=/html/body/div[1]/div[2]/div/div[2]/div[2]/div[2]/div/div@data-column-size  
```

Comment:
This xpath is too big. Any change in html can break it. Can't have a smaller xpath? You can also try using a css selector.

Reply:
Better ?, use xpath, because I had problems with the css selector, if I chose the element, but I could not get the value of an attribute of that element with this method